documentation/docs/examples/create-identity-and-credentials.mdx

```diff
@@ -0,0 +1,78 @@
+# Create Identity and Credentials
+
+The [example-1](https://github.com/iotaledger/integration-services/blob/develop/clients/node/examples/1-CreateIdentityAndCredential.ts)
+script authenticates a Integration Service client to manage Identities using the Admin identity
+created in [example-0](./how-to-run-examples) and then performs the following tasks:
+
+* create a new Identity (`userIdentity`)
+* create a credential for that identity (`userCredential`)
+* verify the credential generated (`await identity.checkCredential(userCredential)`)
+
+You can run the example with the following:
+
+```bash
+npm run example-1
+```
+
+## Authentication
+
+Here's how authenticate the client using the Admin identity and how to get the 
+BasicIdentityCredential that the admin owns: this VC (Verifiable Credential) 
+is required for an Issuer to sign Verifiable Credentials for other Identities.
+
+```js
+import { IdentityClient, IdentityJson } from 'iota-is-sdk';
+
+...
+
+const identity = new IdentityClient(...config...);
+
+// Recover the admin identity
+const adminIdentity = JSON.parse(readFileSync("./adminIdentity.json").toString()) as IdentityJson;
```

Comment:
maybe a check would be useful which then points to example-0 to be executed

Reply:
This guideline is readed sequentially and in the begin of the document we said that "using the Admin identity
created in [example-0](./how-to-run-examples)". I would just make more explicit the example-0 but I already created a page just for setup and example-0 is written in each example. So I would not add any.

Reply:
ok then you can merge it